packages/core/src/components/HelpDrawer/help-drawer.example.html

```diff
@@ -1,24 +1,65 @@
 <div>
-  <p><strong>Note: This is just an example of the HTML markup. See the React example for a functioning example.</strong></p>
-  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+  <p>
+    <strong
+      >Note: This is just an example of the HTML markup. See the React example
+      for a functioning example.</strong
+    >
+  </p>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+    consequat.
+  </p>
   <span class="ds-u-display--block">
     <a href="javascript:void(0);">Toggle the help drawer.</a>
   </span>
   <div class="ds-c-help-drawer">
     <div class="ds-c-help-drawer__header">
-      <div class="ds-u-fill--gray-lightest ds-u-padding--2 ds-u-display--flex ds-u-align-items--start">
-        <h3 tabindex="0" class="ds-u-text--lead ds-u-margin-y--0 ds-u-margin-right--2">Help Drawer Title</h3>
-        <button class="ds-c-button ds-c-button--secondary ds-c-button--small ds-u-margin-left--auto" type="button">Close</button>
+      <div
+        class="ds-u-fill--gray-lightest ds-u-padding--2 ds-u-display--flex ds-u-align-items--start"
+      >
```

Comment:
I personally find this breakdown where the div is broken into 3 lines harder to read, I'm wondering if we can adjust the formatting a bit in a future issue

Reply:
We can adjust it here: https://prettier.io/docs/en/options.html#print-width
would you want to include it in this PR?

Reply:
We can address it later on

Reply:
Would you be opposed to handling it now if its quick? I'm just thinking it will be touching all of these same files again anyway if we bump up the print width to 100 for example

Reply:
I don't know that the print width will really solve the issue. I'm thinking it would crop up again anyways. We can leave it as is for now.